website/docs/configuration/observability.md

```diff
@@ -23,8 +23,11 @@ FerretDB provides the following log formats:
 - `console` is a human-readable format with optional colors;
 - `text` is machine-readable [logfmt](https://brandur.org/logfmt)-like format
   (powered by [Go's `slog.TextHandler`](https://pkg.go.dev/log/slog#TextHandler));
-- `json` if machine-readable JSON format
-  (powered by [Go's `slog.JSONHandler`](https://pkg.go.dev/log/slog#JSONHandler)).
+- `json` is machine-readable JSON format
+  (powered by [Go's `slog.JSONHandler`](https://pkg.go.dev/log/slog#JSONHandler));
+- `mongo` is machine-readable structured JSON format, similar to the one used in MongoDB;
+  It follows the Relaxed Extended JSON specification;
+  Fields required in the output format but not yet implemented, will not be included.
```

Comment:
https://github.com/FerretDB/FerretDB/pull/4716#discussion_r1962030153. Use dots.

Reply:
LGTM, but please resolve that in the next PR, maybe after the `dbStats` issue.